translations/messages+intl-icu.de.yml

```diff
@@ -3110,6 +3110,7 @@ statement.assignment.override: "Stellungnahme mir zuweisen"
 statement.assignment.release: "Stellungnahme freigeben"
 statement.assignment.unassigned: "Aktuell nicht in Bearbeitung"
 statement.attachments.reference: "Referenzen auf Anhänge"
+statement.original.attachment.reference: "Referenzen auf original Anhang"
```

Comment:
Not so clear what this should be. When this should be kept it should be at least `Originalanhang`

Reply:
done in [ba0023b](https://github.com/demos-europe/demosplan-core/pull/2444/commits/ba0023badff5ef07ebf1b134aabd7d377c15dc6e)

Reply:
I need to know the final outcome to adjust this in https://github.com/demos-europe/demosplan-core/pull/2363
pls notify :)

Reply:
I woud then suggest only `Originalanhang`

Reply:
done in [45f9819](https://github.com/demos-europe/demosplan-core/pull/2444/commits/45f98190a2614e931ec22813a601e7051689e075)

Reply:
@MoritzMandler `Originalanhang` it is then